### PR TITLE
Copy-edits

### PR DESCRIPTION
**Pull request by dkumaras**
### Description

Please describe what this change achieves. Ensure you have read the [Contributing to InSpec AWS](https://github.com/inspec/inspec-aws/CONTRIBUTING.md) document before submitting.

### Issues Resolved

List any existing issues this PR resolves, or any Discourse or StackOverflow discussion that's relevant

### Check List
Please fill box or appropriate ([x]) or mark N/A.
- [ ] New functionality includes integration tests/controls
- [ ] New Terraform resources
- [ ] Documentation provided or updated for resources 
- [ ] All Integration Tests pass
- [ ] All Unit Tests pass
- [ ] `rake lint` passes
- [ ] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>

### CODE REVIEW

**Review by ppradhan9 — COMMENTED** [review]
@dkumaras : 
DCO is failing while committing any changes do a signed commit.
git commit -s -m "message"

File: docs/resources/aws_efs_mount_target.md [hunk 1]
```diff
@@ -48,12 +48,12 @@ For additional information, see the [AWS documentation on EFS Mount Target](http
 ### Ensure that the life cycle state is available.
 
     describe aws_efs_mount_target(mount_target_id: 'MOUNT_TARGET_ID') do
-        its('life_cycle_state') { should eq 'available' }
+        its('life_cycle_state') { should eq 'AVAILABLE' }
```
Comment (IanMadd) [inline docs/resources/aws_efs_mount_target.md:51]: Lowercase `available` is correct in this example.

**Comment by IanMadd** [comment]
Closing this one too. Lowercase 'available' is correct, the other changes aren't that significant.

**Comment by dkumaras** [comment]
> @dkumaras :
> DCO is failing while committing any changes do a signed commit.
> git commit -s -m "message"

Sure. Done.

**Pull request closed without merging**